guide/src/class.md

```diff
@@ -618,6 +616,10 @@ TODO: Which traits to implement (basically `PyTypeCreate: PyObjectAlloc + PyType
 
 ## How methods are implemented
 
-Users should be able to define a `#[pyclass]` with or without `#[pymethods]`, while pyo3 needs a trait with a function that returns all methods. Since it's impossible make the code generation in pyclass dependent on whether there is an impl block, we'd need to make to implement the trait on `#[pyclass]` and override the implementation in `#[pymethods]`, which is to my best knowledge only possible with the specialization feature, which is can't be used on stable.
+Users should be able to define a `#[pyclass]` with or without `#[pymethods]`, while PyO3 needs a
+trait with a function that returns all methods. Since it's impossible make the code generation in
```

Comment:
it's impossible make -> it's impossible **to** make ( though it's not your mistake :)